jupyterlab.tera

```diff
@@ -17,7 +18,7 @@ export class CatppuccinPalettes {
   {%- for id, flavor in flavors %}
 
   setColors{{flavor.name | slugify | capitalize}}() {
-    {%- for name, color in flavor.colors -%}
```

Comment:
I know you are running Prettier after generating the templates (though I don't know what `jlpm` is so I commented that out in my testing), but removing this whitespace control means that the Whiskers output is almost identical to the formatted version.

Reply:
FYI (from https://jupyterlab.readthedocs.io/en/stable/developer/contributing.html):
```
The jlpm command is a JupyterLab-provided, locked version of the yarn package manager.
```

Thanks for this small but useful modification. As you said, the output now is almost 100% compliant with the style guides, missing only some lines that exceed 80 characters. 

I would prefer to keep it compatible with Jupyter Lab style guide, so I will avoid personalizing the max characters style settings.